docs/COMPONENTS.md

```diff
@@ -37,6 +37,8 @@ Each EDS component is documented in Storybook, which surfaces each component's A
 
 ### Creating a component
 
+**NOTE**: Refer to this [Pull Request](https://github.com/chanzuckerberg/edu-design-system/pull/1329) for examples of potential upcoming components. If you see one you like, you can use it as a baseline for implementation.
```

Comment:
Could also create a tag or cut a branch before this pr lands, so that it's more easily searchable in the github searchbox as well 
![image](https://user-images.githubusercontent.com/86632227/194426776-1adcf3cd-9089-446e-a0f4-45e68d2a0b61.png)

Reply:
Good idea. I can push a tag, and refer to that here